.mocharc.yml

```diff
@@ -1,2 +1,3 @@
 require: ts-node/register
 spec: './test/**/*.ts'
+timeout: 60000 # same as "timeout: '60s'"
```

Comment:
Could you add the motivation of having this specific value to the comment?

Reply:
Sure. Will do that.